tests/byzantium/eip196_ec_add_mul/test_ecadd.py

```diff
@@ -52,7 +150,40 @@ def test_valid(
             b"",
             id="pt_1_1_plus_inf",
         ),
+        pytest.param(
+            Spec.INF_G1 + PointG1(1, 3),
+            b"",
+            id="inf_plus_pt_1_3",
+        ),
+        pytest.param(
+            PointG1(0, 3) + Spec.INF_G1,
+            b"",
+            id="pt_0_3_plus_inf",
+        ),
+        pytest.param(
+            PointG1(1, 3) + b"\0" * 16,
+            b"",
+            id="pt_1_3_plus_16_zero_bytes",
+        ),
```

Comment:
```suggestion
        ),
        pytest.param(
            PointG1(1, 3) + b"\0" * 32,
            b"",
            id="pt_1_3_plus_32_zero_bytes",
        ),
```
For completion sake, 32 extra zero bytes was in one of the files.